code/acting/src/acting/DummyTrajectorySub.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+"""
+This node tests the subscription side of the dummy trajectory message.
+It therefore receives a nav_msgs/Path msg.
+"""
+
+import ros_compatibility as roscomp
+from ros_compatibility.node import CompatibleNode
+from nav_msgs.msg import Path
+
+
+class DummyTrajectorySub(CompatibleNode):
+    """
+    Creates a node capable of subscribing to a Path msg.
+    """
+
+    def __init__(self):
+        """
+        Constructor
+        :return:
+        """
+
+        super(DummyTrajectorySub, self).__init__('dummy_trajectory_sub')
+        self.loginfo("DummyTrajectorySub node started")
+
+        # basic info
+        self.role_name = self.get_param("role_name", "ego_vehicle")
+        self.control_loop_rate = self.get_param("control_loop_rate", "0.05")
```

Comment:
Shouldn't be the value for control_loop_rate a float instead of a string?

Reply:
It works both ways